packages/react-scripts/package.json

```diff
@@ -50,15 +50,14 @@
     "fs-extra": "^10.0.0",
     "html-webpack-plugin": "5.3.2",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.6.0",
-    "jest-circus": "26.6.0",
-    "jest-resolve": "26.6.0",
-    "jest-watch-typeahead": "0.6.1",
+    "jest": "27.1.0",
+    "jest-resolve": "27.1.0",
+    "jest-watch-typeahead": "0.6.4",
     "mini-css-extract-plugin": "2.1.0",
     "postcss": "8.3.5",
     "postcss-flexbugs-fixes": "5.0.2",
     "postcss-loader": "6.1.1",
-    "postcss-normalize": "10.0.0",
+    "postcss-normalize": "10.0.1",
```

Comment:
for this issue https://github.com/facebook/create-react-app/discussions/11278#discussioncomment-1332294